docs/examples/ButtonGroupBlock.js

```diff
@@ -1,5 +1,5 @@
 const buttonGroupInstance = (
-  <ButtonGroup vertical block>
+  <ButtonGroup vertical style={{ display: 'block' }}>
```

Comment:
maybe `style={{ width: '100%' }}`? or does doing `display: block` actually work for making the group be full-width?

Reply:
`.btn-group-vertical > .btn` already has a default style for being displayed block and 100% width.

`display: block` on the ButtonGroup accomplishes 100% width in a more clear and predictable way than relying on the inherited `display: inline-block` and adding `width: 100%` IMO.

Reply:
ah, cool